### PR TITLE
[vividus] Do not use bdd prefix for variables definitions

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: docs/modules/commons/pages/variables.adoc [hunk 1]
```diff
@@ -65,13 +65,13 @@ NOTE: The scopes names are case-insensitive.
 |Variables are available in the next batches. `next_batches`-scoped variable is not available in batch it's declared within.
 
 |`batch`
-a|Such variables are declared only with `bdd.variables.batch-<batch-number>.` prefix and available throughout the execution of the batch in the suite. An attempt to create the batch-scoped variables during test execution leads to an error.
+a|Such variables are declared only with `variables.batch-<batch-number>.` prefix and available throughout the execution of the batch in the suite. An attempt to create the batch-scoped variables during test execution leads to an error.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:68]: logically it should be `batch-<batch-number>.variables.`, but I would say it's under discussion

Comment (ikalinin1) [reply docs/modules/commons/pages/variables.adoc:68]: Really good point, I don't have objections!

Comment (valfirst) [reply docs/modules/commons/pages/variables.adoc]: also in this case: `variables.global.` -> `variables.` (full sync between suite and batch levels)

File: vividus/src/main/resources/properties/deprecated/depreacted.properties [hunk 2]
```diff
@@ -3,3 +3,5 @@ bdd.story.fail-fast=story.fail-fast
 bdd.scenario.fail-fast=scenario.fail-fast
 bdd.story-loader.batch-(\\d+)(.+)=batch-$1$2
 bdd.batch-(\\d+)(.+)=batch-$1$2
+bdd.variables.global(.+)=variables$1
```
Comment (valfirst) [inline vividus/src/main/resources/properties/deprecated/depreacted.properties:6]:
```suggestion
bdd.variables.global.(.+)=variables.$1
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus/src/main/resources/properties/deprecated/depreacted.properties (hunk 2 above)
Comment (uarlouski) [reply vividus/src/main/resources/properties/deprecated/depreacted.properties:6]: btw should not we escape dots?

Comment (valfirst) [reply vividus/src/main/resources/properties/deprecated/depreacted.properties:6]: yes, good cacth

**Review by valfirst — COMMENTED** [review]
`../depreacted.properties` -> `../deprecated.properties`

**Pull request merged**